docs/concepts.rst

```diff
@@ -202,8 +202,8 @@ Caching and Protocols
 Whenever Bionic computes an entity's value, it automatically caches that value
 in memory (in case you access it again in from the same ``Flow`` object) and in
 persistent storage (in case you want to access it later, perhaps after
-restarting your script or notebook).  Currently the only supported location for
-persistent storage is your computer's hard disk.
+restarting your script or notebook).  Currently the supported locations for
```

Comment:
Reading this paragraph again, maybe we should just replace "in persistent storage" with "to disk", and then mention Google Cloud Storage in the next sentence. BTW, when we mention GCS we can include a link to the section in the doc that addresses it.